src/test/scala/uk/gov/homeoffice/cirium/CiriumSpec.scala

```diff
@@ -134,6 +137,13 @@ class CiriumSpec extends TestKit(ActorSystem("testActorSystem", ConfigFactory.em
     result === expected
   }
 
+  "I should get exception while parsing response that does not have request object" >> {
+    val expected = CiriumFlightStatusResponseFailure(DeserializationException("Object is missing required member 'request'"))
```

Comment:
Shall we change this `CiriumFlightStatusResponseFailure` to an object and have the test check the type rather than the string contents of an exception?